pyproject.toml

```diff
@@ -5,9 +5,6 @@ build-backend = "setuptools.build_meta"
 [tool.setuptools_scm]
 write_to = "pyscreener/_version.py"
 
-[tool.pytest.ini_options]
-addopts = "--cov pyscreener"
-
```

Comment:
why is this removed?

Reply:
My apologies, I'd removed that during testing as I was having issues with pytest. Never meant for it to be committed which shows the value of the PR system I guess!

I've updated the readme to specify pip installation of `pytest-cov` instead of just `pytest` as the standard version doesn't conduct coverage checks.